kolibri/plugins/coach/assets/src/vue/groups-page/delete-group-modal.vue

```diff
@@ -0,0 +1,70 @@
+<template>
+
+  <core-modal :title="$tr('deleteLearnerGroup')"
+    @cancel="close">
+    <p>{{ $tr('areYouSure') }} <strong>{{ groupName}}</strong>?</p>
```

Comment:
String won't get translated properly in many languages here, especially with the `?` hanging outside of the string itself. 

I don't actually have a good solution to this. I've run into this issue a good amount of times now, and I end up forgoing the `<strong>` styling pretty much every time. 

I'd talk to @rtibbles to see if he's got any creative solutions, or talk to the design team to see if the styling is vital. This feature's going to be used in lots of non-english languages, so I think this'd be kind of important.